tests/robot-tests/tests/admin_and_public/bau/publish_release_and_amend.robot

```diff
@@ -315,8 +315,8 @@ Navigate to data replacement page
     user checks headed table body row contains    Subject title    Dates test subject
     user checks headed table body row contains    Data file    dates.csv
     user checks headed table body row contains    Metadata file    dates.meta.csv
-    user checks headed table body row contains    Number of rows    118
-    user checks headed table body row contains    Data file size    17 Kb
+    user checks headed table body row contains    Number of rows    118    wait=%{WAIT_SMALL}
```

Comment:
Is there something about these cells specifically that require the WAITs?  Are they lazy?

Reply:
No, they aren't lazy loaded. The way the backend returns this information has changed which I think has introduced a bit of flakiness in this area (see https://dfedigital.atlassian.net/browse/EES-3571, https://github.com/dfe-analytical-services/explore-education-statistics/pull/3387) I noticed that tests that check file sizes and rows became flaky. They now transition from 'Unknown' > '<file size>' which may be the cause.